tests/test_model.py

```diff
@@ -837,7 +837,7 @@ def test_update(self, mock_time):
         """
         mock_time.side_effect = [1559692800]  # 2019-06-05 00:00:00 UTC
         self.init_table_meta(SimpleUserModel, SIMPLE_MODEL_TABLE_DATA)
-        item = SimpleUserModel('foo', is_active=True, email='foo@example.com', signature='foo')
+        item = SimpleUserModel(user_name='foo', is_active=True, email='foo@example.com', signature='foo', views=100)
```

Comment:
Note that this [assertion](https://github.com/pynamodb/PynamoDB/blob/master/tests/test_model.py#L915) always passed since `views` was never set.